src/pages/guides/using/asset-events-actions.md

```diff
@@ -0,0 +1,514 @@
+
+---
+
+#### Created Manifest
+
+An Event is triggered by the upload of a [Manifest Resource](asset-events-glossary.md#manifest) to a Composite. In `event:resources`, the [Repository Metadata Resource](asset-events-glossary.md#repository-metadata-resource) is listed as `updated`. The following [resources](asset-events-glossary#resource) are listed as `created`:
+
+- [Primary](asset-events-glossary.md#primary-resource)
+- [Manifest](asset-events-glossary.md#manifest)
+- [Embedded Metadata](asset-events-glossary.md#embedded-metadata-resource)
+- [Versions](asset-events-glossary.md#versions-resource)
+- [Rendition](asset-events-glossary.md#rendition-resource)
+
+```json
+{
+  "body": {
+    "xdmEntity": {
+      "event:resources": {
+        "http://ns.adobe.com.adobecloud/rel/metadata/repository": {
+          "event:action": "updated"
+        },
+        "http://ns.adobe.com/adobecloud/rel/primary": {
+          "event:action": "created"
+        },
+        "http://ns.adobe.com.adobecloud/rel/manifest": {
+          "event:action": "created"
+        },
+        "http://ns.adobe.com/adobecloud/rel/metadata/embedded": {
+          "event:action": "created"
+        },
+        "version-history": {
+          "event:action": "created"
+        },
+        "http://ns.adobe.com/adobecloud/rel/rendition": {
+          "event:action": "created"
+        }
+      }
+    }
+  }
+}
+```
+
+---
+
+#### Updated Manifest
+
+An Event is triggered by the update of the [Manifest Resource](asset-events-glossary.md#manifest) of a [Composite](asset-events-glossary#composite). In `event:resources`, the following [resources](asset-events-glossary#resource) are listed as `updated`:
+
+- [Repository Metadata](asset-events-glossary.md#repository-metadata-resource)
+- [Primary](asset-events-glossary.md#primary-resource)
+- [Manifest](asset-events-glossary.md#manifest)
+- [Embedded Metadata](asset-events-glossary.md#embedded-metadata-resource)
+- [Versions](asset-events-glossary.md#versions-resource)
+- [Rendition](asset-events-glossary.md#rendition-resource)
+
+```json
+{
+  "body": {
+    "xdmEntity": {
+      "event:resources": {
+        "http://ns.adobe.com.adobecloud/rel/metadata/repository": {
+          "event:action": "updated"
+        },
+        "http://ns.adobe.com/adobecloud/rel/primary": {
+          "event:action": "updated"
+        },
+        "http://ns.adobe.com.adobecloud/rel/manifest": {
+          "event:action": "updated"
+        },
+        "http://ns.adobe.com/adobecloud/rel/metadata/embedded": {
+          "event:action": "updated"
+        },
+        "version-history": {
+          "event:action": "updated"
+        },
+        "http://ns.adobe.com/adobecloud/rel/rendition": {
+          "event:action": "updated"
+        }
+      }
+    }
+  }
+}
+```
+
+---
+
+#### Created Component
```

Comment:
We can get rid of the **"Created Component"** and **"Updated Component"** examples because we don't listen to these events. A composite event gets triggered when the manifest corresponding to that composite is created/updated.